### DISCUSSION TITLE
Kubernetes v1.26 Feature Blogs publication order and dates

### DISCUSSION THREAD

**Discussion by fsmunoz** [kubernetes/sig-release#2105] [General]
For v1.26 we are aiming at **4 blog posts per week** (every day, with an interval in the middle).

There are [14](https://github.com/orgs/kubernetes/projects/98/views/11) Ready for Review feature blogs, plus an additional one that might not fit in the series.

Considering this, and after analysing the Major Themes, plus some balance around SIGs, this is the proposal

| Title                                                           | Link                                             | Order | Date       |
|-----------------------------------------------------------------|--------------------------------------------------|-------|------------|
| Signing Release Artifacts                                       | https://github.com/kubernetes/website/pull/37639 |     1 | 12/12/2022 |
| Support for Windows privileged containers                       | https://github.com/kubernetes/website/pull/37370 |     2 | 13/12/2022 |
| dynamic resource allocation                                     | https://github.com/kubernetes/website/pull/37434 |     3 | 15/12/2022 |
| Non-graceful node shutdown                                      | https://github.com/kubernetes/website/pull/37583 |     4 | 16/12/2022 |
| Graduate DeviceManager to GA                                    | https://github.com/kubernetes/website/pull/37466 |     5 | 19/12/2022 |
| CEL for Admission Control                                       | https://github.com/kubernetes/website/pull/38162 |     6 | 20/12/2022 |
| Kubelet Credential Provider                                     | https://github.com/kubernetes/website/pull/37647 |     7 | 22/12/2022 |
| Allow Kubernetes to supply pod's fsgroup to CSI driver on mount | https://github.com/kubernetes/website/pull/37621 |     8 | 23/12/2022 |
| Pod Scheduling Readiness                                        | https://github.com/kubernetes/website/pull/37436 |     9 | 26/12/2022 |
| Graduate to CPUManager to GA                                    | https://github.com/kubernetes/website/pull/37435 |    10 | 27/12/2022 |
| Job tracking without lingering Pods                             | https://github.com/kubernetes/website/pull/37438 |    11 | 29/12/2022 |
| Service Internal Traffic Policy                                 | https://github.com/kubernetes/website/pull/37648 |    12 | 30/12/2022 |
| Provision volumes from cross-namespace snapshots                | https://github.com/kubernetes/website/pull/37628 |    13 |   2/1/2023 |
| Allow StatefulSet to control start replica ordinal numbering    | https://github.com/kubernetes/website/pull/37418 |    14 |   3/1/2023 |
| Retroactive default StorageClass assignement                    | https://github.com/kubernetes/website/pull/37625 |    15 |   5/1/2023 |
| PodHealthyPolicy for PodDisruptionBudget                        | https://github.com/kubernetes/website/pull/37627 |       |   6/1/2023 |

Should we account for the festivities in some way? Pausing one week (like the last week of December) is a possibility, but OTOH it might be unnecessary (considering that the publishing process is automated).

**Comment by saschagrunert** [comment]
Changed https://github.com/kubernetes/website/pull/37639 to point to Dec 12.

**Reply by fsmunoz** [reply]
Hi @saschagrunert , that should work, but if there is any change I will update you (I didn't yet posted in every PR because I wanted to get more input for the SIG Docs theme in terms of the starting date and pacing).

**Comment by fsmunoz** [comment]
Slight adjustments after reviewing with @sftim .

**Comment by fsmunoz** [comment]
Added untracked blog that was being reviewed but didn't communicate opt-in.

**Comment by sftim** [comment]
> Should we account for the festivities in some way? 

I'd skip the 25th of December and the 1st of January as these are end-of-year festivals in countries that represent a lot of contributors. However, as we're a global project, we shouldn't give undue weight to one culture over others (for example, we might want to also avoid [lunar new year](https://en.wikipedia.org/wiki/Lunar_New_Year)).

**Reply by fsmunoz** [reply]
Agreed, fortunately neither happened to be one of the dates, which is why I didn't really press that idea.
